js/index.js

```diff
@@ -25,4 +25,120 @@ menuIcon.addEventListener('click', openMenuModal);
+  },
+  {
+    id: 3,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-3.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+  {
+    id: 4,
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    image: { src: '../assets/snapshoot-4.png', title: 'Portfolio' },
+    meta: ['CANOPY', 'Back End Dev', '2015'],
+    technologies: ['html', 'css', 'javascript'],
+    sourceLink: 'https://github.com/BB-Simon',
+    liveLink: 'https://bb-simon.github.io/',
+  },
+];
+
+// Listener's functions
+const createProjectCard = (project) => {
+  const cardContainer = document.createElement('div');
+  if (project.id === 1 || project.id === 3) {
+    cardContainer.classList.add('card-conntainer', 'flex-reverse');
+  } else {
+    cardContainer.classList.add('card-conntainer');
+  }
+
+  const techList = project.technologies.map((tech) => `<li class="card-category">${tech}</li>`);
+  const metaList = project.meta.map((meta, index) => {
+    if (index !== 0) {
+      return `<div class="divider"></div>
+        <span class="card-meta-item muted">${meta}</span>`;
+    }
+    return `<span class="card-meta-item">${meta}</span>`;
+  });
+
+  const html = `
+    <img src=${project.image.src} alt=${project.image.title}>
+    <div class="card-content">
+        <h2 class="card-title">${project.name}</h2>
+        <div class="card-meta">
+          ${metaList.join('')}
+        </div>
+        <p id="card-description">${project.description}</p>
+        <ul class="card-categories-wrapper">
+          ${techList.join('')}
+        </ul>
+        <div class="card-cta-wrapper">
+            <button class="card-btn see-project">See Project</button>
+        </div>
+    </div>`;
+  cardContainer.innerHTML = html;
+  worksSection.appendChild(cardContainer);
+};
+
+projectData.forEach((project) => createProjectCard(project));
+const cardBtn = document.querySelectorAll('.see-project');
+const modalOverlay = document.getElementById('project-detail-modal-overlay');
+const projectModal = document.getElementById('project-detail-modal');
+const projectCloseBtn = document.getElementById('project-close-btn');
+
+const openProjectModal = () => {
+  const scrollBarWidth = window.innerWidth - document.body.offsetWidth;
+  document.body.style.margin = `0px ${scrollBarWidth}px 0px 0px`;
+  document.body.style.overflow = 'hidden';
+  modalOverlay.style.display = 'block';
+  projectModal.style.display = 'block';
+};
+const closeProjectModal = () => {
+  document.body.style.margin = '';
+  document.body.style.overflow = '';
+  modalOverlay.style.display = 'none';
+  projectModal.style.display = 'none';
+};
+
```

Comment:
- [ ] Please make sure that when one clicks on the `see project` button to see a project, the same project modal should show but not a different project modal, as you can see from the video below some projects are showing different project modals.


https://user-images.githubusercontent.com/92303891/209418911-9e0f492f-d233-4463-9c78-acb84ab7df65.mp4